### DISCUSSION TITLE
What is the diffrence between this project and p4vfs from Perforce?

### DISCUSSION THREAD

**Discussion by ArvoX** [microsoft/p4vfs#40] [Q&A]
Both this project and Perforce has a p4vfs client but is there any driffences? I have not found any sources compairing or talk about the 2 clients at the same time.

Link to Perforces version: https://www.perforce.com/downloads/helix-core-virtual-file-service

**Comment by jessk-msft** [comment]
This project [Microsoft P4VFS](https://github.com/microsoft/p4vfs) pre-dates and is completely unrelated to Helix Core Virtual File Service.
